spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../person'
+
+# rubocop:disable Lint/UselessAssignment
+
+describe Person do
+  before :each do
+    @person = Person.new(6, 'Milu', parent_permission = true)
+  end
+
+  it 'Should be return name Milu' do
+    expect(@person.name).to eq 'Milu'
+  end
+
+  it 'Should be return age 6' do
+    expect(@person.age).to eq 6
+  end
+
+  it 'Should be return parent permission true' do
+    expect(@person.parent_permission).to eql true
+  end
+end
+
+# rubocop:enable Lint/UselessAssignment
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_rental,  of_age?,   and  correct_name` class person method has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end

  def correct_name
    @name
  end

  private

  def of_age?
    @age >= 18
  end
```

- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
Already implemented it! Thanks for the feedback! :smile:

Reply:
Already implemented it! Thanks for the feedback! :smile: